### PR TITLE
Styling: adding bold font weight definition

### LINKED ISSUE #12391: Missing bold font-face declaration
`_registerFontFaceSet()` does not add a declaration for `bold`, only `light`, `semilight`, `regular` and `semibold`:

https://github.com/microsoft/fluentui/blob/f9ca45d012d2e77eea728e19651b144452562004/packages/styling/src/styles/DefaultFontStyles.ts#L27

### Environment Information

- **Package version(s)**: 7.105.2
- **Browser and OS versions**: Chrome 80.0.3987.149, Mac OSX Catalina

### Please provide a reproduction of the bug in a codepen:
```ts
let { 
  Fabric 
} = window.Fabric;

const Content = () => {
  return (
    <Fabric>
      <div style={{fontWeight: 100}}>
        Hello World
      </div>
      <div style={{fontWeight: 300}}>
        Hello World
      </div>
      <div style={{fontWeight: 400}}>
        Hello World
      </div>
      <div style={{fontWeight: 600}}>
        Hello World
      </div>
      <div style={{fontWeight: 700}}>
        Hello World
      </div>
    </Fabric>
  );
};

ReactDOM.render(
  <Content />, 
  document.getElementById("content")
);

```

#### Actual behavior:
Bold (700) font weight looks the same as semibold (600), at least in the Chrome browser. IE11 might be stretching the semibold font to render it more bold.

#### Expected behavior:

Bold 700 font-weight renders the correct weight.

### Priorities and help requested:

Are you willing to submit a PR to fix? Maybe

Requested priority: Normal

Products/sites affected: Any web app that does not separately include Fabric CSS with additional font overrides.
https://www.yammer.com
https://fabricweb.z5.web.core.windows.net/oufr/7.105.2/

### PR DESCRIPTION
**Pull request by dzearing**
#### Pull request checklist
- [x] Addresses an existing issue: Fixes #12391
- [x] Include a change request file using `$ yarn change`

It seems the bold font weight has been missing for some time from the font-face definitions.

Before:
image

After:
image

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/fluentui/pull/12398)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/Text/examples/Text.Weights.Example.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { Text } from 'office-ui-fabric-react/lib/Text';
```
Comment (aneeshack4) [inline packages/office-ui-fabric-react/src/components/Text/examples/Text.Weights.Example.tsx]: Shouldn't this example show up in the PR deploy site? http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/12398/merge/fabric-website/dist/index.html?devhost#/controls/web/text

File: apps/vr-tests/src/stories/Fonts.stories.tsx [hunk 2]
```diff
@@ -43,6 +45,15 @@ storiesOf('Fonts', module)
       {story()}
     </Screener>,
   )
+  .addStory('Weights', () => (
```
Comment (aneeshack4) [inline apps/vr-tests/src/stories/Fonts.stories.tsx]: Looks great!
image

File: packages/office-ui-fabric-react/src/components/Text/examples/Text.Weights.Example.tsx (hunk 1 above)
Comment (aneeshack4) [reply packages/office-ui-fabric-react/src/components/Text/examples/Text.Weights.Example.tsx]: Might need to add it to Text.doc.tsx

**Comment by aneeshack4** [comment]
Where are the screenshots in the description coming from? I tried our typography page on Mac - Chrome & Windows - Edge & on both, the semibold (600) and bold (700) are not identical (whereas they are, incorrectly, when the font weights are actually used like in the codepen). 
image

The [Typography page code](https://github.com/microsoft/fluentui/blob/master/apps/fabric-website/src/pages/Styles/TypographyPage/TypographyPage.tsx#L130) seems to be getting the [weights](https://github.com/microsoft/fluentui/blob/master/apps/fabric-website/src/pages/Styles/TypographyPage/TypographyPage.tsx#L46) from [somewhere else](https://github.com/microsoft/fluentui/blob/master/apps/fabric-website/src/pages/Styles/TypographyPage/TypographyPage.tsx#L16) - perhaps that's why. 

@swese44 please feel free to look over this PR too.

**Comment by dzearing** [comment]
@aneeshack4 Was on the storybook examples. I don't really want to add the examples to public docs.

**Pull request merged**